### ISSUE TITLE
Training on the MVSEC dataset

### ISSUE THREAD

**Issue by ZPJ-LEFT** [danqu130/DCEIFlow#5]
Your work is very meaningful! 
After reading paper and codes, I am confused about the contents in Table IV.  Table IV show the results of two baselines and DCEIFlow on the MVSEC dataset. My confusion is how you trained models on MVSEC:

1. How the training sequence was divided? The 'valid_time_index ' of outdoor_day2 is set to be [4375,7002]. Should I only use them to train?
2. Which is the Epoch to be set? 200 or 300? And how the other hyperparameters configured?
3. Can you release the weights of DCEIFlow trained on MVSEC?

I am looking forward to your reply.

**Comment by danqu130** [comment]
Thank you for your interest in our work. I hope I can solve your concerns.

1. The training data for the model trained with MVSEC are all valid indexs of outdoor_day2, i.e., [1950, 28500].
2. The experiments in our paper are training 100 epochs on FlyingChairs2 and 200 epochs on MVSEC. For MVSEC, batch_size is 16, and the other hyperparameters remain unchanged.
3. Since a longer time has passed, I will try to find it.

I apologize for the unclear expressions in the paper. I think I need to clarify some additional experimental details. 

The experiments on the MVSEC dataset only were additionally requested by the reviewers and were not the focus of our paper. Therefore we made a fair comparison with baselines only and not with existing methods in Table 4. In the experiments on MVSEC, we not only used the full outdoor_day2 data, but also add image data augmentation as well as an unsupervised training loss function (cf. ARFlow), in order to achieve acceptable generalization performances. The weights of supervised and unsupervised losses are 1:2.

**Comment by ZPJ-LEFT** [comment]
Thanks for your reply. It is very helpful to me!
